benchmarks/src/tpch/run.rs

```diff
@@ -92,6 +92,15 @@ pub struct RunOpt {
     #[structopt(short = "j", long = "prefer_hash_join", default_value = "true")]
     prefer_hash_join: BoolDefaultTrue,
 
+    /// If true then Piecewise Merge Join can be used, if false then it will opt for Nested Loop Join
+    /// True by default.
+    #[structopt(
+        short = "j",
```

Comment:
I believe this causes a bug as the `-j` is duplicated with `prefer_hash_join` above. 

Here is a PR to fix:
- https://github.com/apache/datafusion/pull/19033